api/src/main/java/io/minio/S3Escaper.java

```diff
@@ -19,9 +19,14 @@
 import com.google.common.escape.Escaper;
 import com.google.common.net.UrlEscapers;
 
-class S3Escaper {
+public class S3Escaper {
+
   private static final Escaper ESCAPER = UrlEscapers.urlPathSegmentEscaper();
 
+  private S3Escaper() {
+    // no instances plz
```

Comment:
```suggestion
    // no instances please
```

Reply:
I don't think we need this comment.

Reply:
sorry, it's a fight with warnings in my workspace  =)

Reply:
We have many such constructors. Can't you adjust your editor more precisely to google java format?

Reply:
It's not a spotless/checkstyle issues, that can use a google style guide. It's sonar issues =). I've changed it.